src/components/FormPanel/FormPanel.tsx

```diff
@@ -772,13 +773,12 @@ export const FormPanel: React.FC<Props> = ({
    * Check updated values
    */
   const isUpdated = useMemo(() => {
-    for (const element of elements) {
-      if (element.value !== initial[element.id]) {
-        return true;
-      }
-    }
-
-    return false;
+    return elements
+      .map((element) => {
```

Comment:
please revert to the previous version with `for` and just update `if` statement

Reply:
done